### DISCUSSION TITLE
:tada: RGBWarp (WIP)

### DISCUSSION THREAD

**Discussion by jgarza9788** [Schneegans/Burn-My-Windows#483] [Ideas]
hi me again ... still helping out with these effects

based on https://www.shadertoy.com/view/4cdfWl
code at https://github.com/jgarza9788/Burn-My-Windows/tree/RGBWarp

Merry Christmas 

https://github.com/user-attachments/assets/8943bc19-1a34-44cc-aedb-f4d36d0d455e

**Comment by Schneegans** [comment]
Merry Christmas!

This looks very cool! I'm looking forward to some new PRs :smile:!
